events-api/src/modules/payments/payments.module.ts

```diff
@@ -0,0 +1,10 @@
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+import { PaymentsService } from './payments.service';
+
+@Module({
+  providers: [PaymentsService],
+  imports: [HttpModule],
```

Comment:
You can configure HttpModule with help of Config Module

Reply:
You can pass base URL, headers and so on

Reply:
> You can pass base URL, headers and so on

fixed